### DISCUSSION TITLE
If we want people to post discussion threads, they need to have a GitHub Account

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-website-jekyll#28] [Riverscapes Data Exchange] [answered]
@MattReimer, I like that we funnel people from WebRAVE to this discussion forum. However, we didn't anticipate that if they don't have a GitHub handle/account, they can't post.  We may want to warn them they need to have an account before they can post?

![image](https://user-images.githubusercontent.com/9723905/144103869-faf540c1-bf33-45f1-9f90-adb7df7b889c.png)

I just tried sending Steve Bennett there and it took him 5 minutes to get a damn account. Just worth thinking if we can help them along.

**Comment by MattReimer** [comment, marked as answer]
Not a bad idea. Adding a bit of text in there is easy so we should do it. I made an issue:

- https://github.com/Riverscapes/rs-web-monorepo/issues/213

so we'll get that into whatever the next deployment is
